fn/develop/configs.md

```diff
@@ -1,22 +1,115 @@
-# Function Config Vars
+# Use of environment variables
 
-There are three ways to get configuration variables into a function, all of which will be available as environment variables.
-These are ordered in order of preference, the later ones overriding the previous ones.
+If you want to use configuration variables into a function and make them available as environment variables, then you can set them using the Fn tool or the func.yaml file.
+ 
+By using the Fn tool, you can set configuration variables on the application or configuration level.
```

Comment:
Suggested change:

By using the Fn CLI, you can set configuration variables on the application or function level.

Reply:
Done